core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/datalake/upgrade/validation/ClusterUpgradeImageValidationEvent.java

```diff
@@ -35,4 +35,12 @@ public CloudContext getCloudContext() {
         return cloudContext;
     }
 
+    @Override
+    public String toString() {
+        return "ClusterUpgradeImageValidationEvent{" +
+                "cloudCredential=" + cloudCredential +
```

Comment:
Have you checked if we're logging any sensitive here?

Reply:
haven't until now: https://github.com/hortonworks/cloudbreak/blob/master/cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/CloudCredential.java#L60
so it should be safe